### DISCUSSION TITLE
Editing lane markings

### DISCUSSION THREAD

**Discussion by evoevo07** [carla-simulator/carla#7861] [Q&A]
Hey guys,
I have built Carla from source and am currently playing around a bit in the UE editor. For my project, it is important to change the road markings (color, dashed, solid, etc.). Does anyone know an efficient method to do this?
Thank you in advance.

**Comment by AinaRoca** [comment]
Hello,
To change the color and other properties like dirt, you can modify the material instance.
But to move or change the lane markings from dashed to solid, you will have to redo the roads using RoadRuner since our lane markings and the road are two separate meshes that fit each other, with their own material instances.
